docs/pages/development/installation.md

```diff
@@ -156,16 +140,15 @@ import App from "./App";
 
 ### Loading published updates
 
-Development builds can also open and preview published updates to your app. To add this feature, add `expo-updates@0.8.0` or newer to your app if it isn't already installed, and add a small additional integration in your **AppDelegate.m** and **MainApplication.java** files.
+Development builds can also open and preview published updates to your app. To add this feature, add `expo-updates@0.13.2` or newer to your app if it isn't already installed, and add a small additional integration in your **AppDelegate.m** and **MainApplication.java** files.
```

Comment:
we should get rid of the version here so we don't have to update it between every release